pkgs/development/python-modules/notebook/default.nix

```diff
@@ -21,32 +22,39 @@
 
 buildPythonPackage rec {
   pname = "notebook";
-  version = "5.0.0";
+  version = "5.2.0";
   name = "${pname}-${version}";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1cea3bbbd03c8e5842a1403347a8cc8134486b3ce081a2e5b1952a00ea66ed54";
+    sha256 = "1sh3jkkmjzv17c3r8ii3kfhpxi6dkjk846b2lfy71g9qwqdcvbvz";
   };
 
-  LC_ALL = "en_US.UTF-8";
+  LC_ALL = "en_US.utf8";
```

Comment:
Is `en_US.utf8` a valid identifier?

Reply:
$ locale -a    
C
en_GB.utf8
en_US.utf8
POSIX

Reply:
Interesting on ubuntu/archlinux always the first notation is used. Python seems to accept both variants:

```console
$ LC_ALL=en_US.utf8 python -c 'import locale; print(locale.getdefaultlocale())'
('en_US', 'UTF-8')
$ LC_ALL=en_US.UTF-8 python -c 'import locale; print(locale.getdefaultlocale())'
('en_US', 'UTF-8')
```